discretize/base/base_mesh.py

```diff
@@ -416,38 +504,42 @@ def copy(self):
         return properties.copy(self)
 
     axis_u = properties.Vector3(
-        'Vector orientation of u-direction. For more details see the docs for the :attr:`~discretize.base.BaseMesh.rotation_matrix` property.',
-        default='X',
-        length=1
+        "Vector orientation of u-direction. For more details see the docs for the :attr:`~discretize.base.BaseMesh.rotation_matrix` property.",
+        default="X",
+        length=1,
     )
     axis_v = properties.Vector3(
-        'Vector orientation of v-direction. For more details see the docs for the :attr:`~discretize.base.BaseMesh.rotation_matrix` property.',
-        default='Y',
-        length=1
+        "Vector orientation of v-direction. For more details see the docs for the :attr:`~discretize.base.BaseMesh.rotation_matrix` property.",
+        default="Y",
+        length=1,
     )
     axis_w = properties.Vector3(
-        'Vector orientation of w-direction. For more details see the docs for the :attr:`~discretize.base.BaseMesh.rotation_matrix` property.',
-        default='Z',
-        length=1
+        "Vector orientation of w-direction. For more details see the docs for the :attr:`~discretize.base.BaseMesh.rotation_matrix` property.",
+        default="Z",
+        length=1,
     )
 
     @properties.validator
     def _validate_orientation(self):
         """Check if axes are orthogonal"""
-        if not (np.abs(self.axis_u.dot(self.axis_v) < 1e-6) and
-                np.abs(self.axis_v.dot(self.axis_w) < 1e-6) and
-                np.abs(self.axis_w.dot(self.axis_u) < 1e-6)):
-            raise ValueError('axis_u, axis_v, and axis_w must be orthogonal')
+        if not (
+            np.abs(self.axis_u.dot(self.axis_v) < 1e-6)
```

Comment:
here, it would be nice to set `tol = 1e-6` (I think it can stay inside the function, but just using a named variable for clarity of code)

Reply:
+1 for code clarity